PythonApp2App/PythonApp2App.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: default
+title: Python App Service Sample
+permalink: /en-US/win10/samples/PythonApp2App.htm
+lang: en-US
+---
+
+## Python App Service Sample
+
+{% include VerifiedVersion.md %}
+
+When writing a Python app for Windows 10 IoT Core you may find yourself trying to use a class available in the UWP namespace only to
+realize that the UWP namespace is not accessible. Although you cannot call UWP functions directly from Python, you still can take advantage 
+of UWP functions. Windows 10 introduced new ways for applications to communicate with each other. One of those ways is to implement 
+"App Services." App Services is a request/response model where one app can call a service located within another app. 
+App Services enables communication between apps, but also with the system. 
+
+### Set up your PC
+* Follow the instructions [here]({{site.baseurl}}/{{page.lang}}/GetStarted.htm) to setup your device and PC.
```

Comment:
Question: Since this is not the content repo, what does 'site.baseurl' point to?